model/VocabularyConfig.php

```diff
@@ -388,6 +388,15 @@ public function getMarcSourceCode($lang = null)
         return $this->getLiteral('skosmos:marcSourceCode', false, $lang);
     }
 
+    /**
+     * Returns a boolean value set in the config.ttl config.
```

Comment:
The docstring should be changed to reflect the meaning of the method - something like "Returns the boolean value of the skosmos:showNotationAsProperty setting".

(I see now that there are other methods with similarly uninformative docstrings in the same class - that's bad, but not a reason to not do it right this time)

Reply:
Done